baca2PackageManager/judge.py

```diff
@@ -0,0 +1,139 @@
+
+from typing import Any
+from enum import Enum
+from abc import ABC, abstractmethod
+
+
+class JudgeVerdict(Enum):
+    OK = 0
+    FAIL = 1
+    INCONCLUSIVE = 2
+
+
+class JudgeNodeBase(ABC):
+
+    def __init__(self, name: str):
+        self._id = name
+
+    def __hash__(self):
+        return hash(self.name)
+
+    @property
+    def name(self) -> str:
+        """:returns: a string value used for hashing an instance"""
```

Comment:
To nie jest komentarz spójny z naszym sposobem komentowania - można go póki co wyrzucić, ale w przypadku klas abstrakcyjnych lepiej jest nawet wszystkie dobrze opisać, żeby przy imporcie pokazywało się w BaCy.